pipelines/candlepin/03-tests.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: run tests
+  become: True
+  hosts:
+   - "{{ forklift_name }}"
+  vars_files:
+    - ../vars/forklift_candlepin.yml
+  tasks:
+    - name: Run acceptance tests
+      ansible.builtin.include_role:
+        name: beaker
+        tasks_from: test
+      vars:
+        beaker_puppet_module: "puppet-candlepin"
+        beaker_os: "{{ pipeline_os | regex_replace('-stream', '') }}"
+        beaker_environment:
+          BEAKER_FACTER_CANDLEPIN_VERSION: "{{ pipeline_version }}"
```

Comment:
one thing that caught me is that *today* this is the Katello version, not the Candlepin version, so when I ran this with `4.3` it installed Candlepin from Katello 4.3, which was 4.1.something and that didn't like Java 17 at all XD

works fine with `nightly`.

Reply:
That's cause the other piece of the puzzle is being able to set the baseurl on the puppet-module -- https://github.com/theforeman/puppet-candlepin/pull/249